// lib0/dist/index.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var array = require('./array-f0f52786.cjs');
var binary = require('./binary-ac8e39e2.cjs');
var broadcastchannel = require('./broadcastchannel-b12b5283.cjs');
var buffer = require('./buffer-c775e94f.cjs');
var conditions = require('./conditions-f5c0c102.cjs');
var decoding = require('./decoding-a2e1942e.cjs');
var diff = require('./diff-6c8c103c.cjs');
var dom = require('./dom-16daf1a0.cjs');
var encoding = require('./encoding-cbd8e85d.cjs');
var environment = require('./environment-a22d11e0.cjs');
var error = require('./error-8582d695.cjs');
var eventloop = require('./eventloop-b299a889.cjs');
var _function = require('./function-a29189eb.cjs');
var indexeddb = require('./indexeddb-0cbb4d6f.cjs');
var iterator = require('./iterator-9fc627c1.cjs');
var json = require('./json-092190a1.cjs');
var log = require('lib0/logging');
var map = require('./map-0dabcc55.cjs');
var math = require('./math-08e068f9.cjs');
var mutex = require('./mutex-63f09c81.cjs');
var number = require('./number-466d8922.cjs');
var object = require('./object-315a738b.cjs');
var pair = require('./pair-ab022bc3.cjs');
var prng = require('./prng-44216e7f.cjs');
var promise = require('./promise-7d13a97c.cjs');
var set = require('./set-a0a3ea69.cjs');
var sort = require('./sort-b8702761.cjs');
var statistics = require('./statistics-c2316dca.cjs');
var string = require('./string-3cd7a2db.cjs');
var symbol = require('./symbol-9c439012.cjs');
var time = require('./time-bc2081b9.cjs');
var tree = require('./tree-9f3c8837.cjs');
var websocket = require('./websocket-3a481d20.cjs');
require('./storage.cjs');
require('./traits.cjs');
require('./metric.cjs');
require('./observable.cjs');

function _interopNamespace(e) {
  if (e && e.__esModule) return e;
  var n = Object.create(null);
  if (e) {
    Object.keys(e).forEach(function (k) {
      if (k !== 'default') {
        var d = Object.getOwnPropertyDescriptor(e, k);
        Object.defineProperty(n, k, d.get ? d : {
          enumerable: true,
          get: function () { return e[k]; }
        });
      }
    });
  }
  n["default"] = e;
  return Object.freeze(n);
}

var log__namespace = /*#__PURE__*/_interopNamespace(log);



exports.array = array.array;
exports.binary = binary.binary;
exports.broadcastchannel = broadcastchannel.broadcastchannel;
exports.buffer = buffer.buffer;
exports.conditions = conditions.conditions;
exports.decoding = decoding.decoding;
exports.diff = diff.diff;
exports.dom = dom.dom;
exports.encoding = encoding.encoding;
exports.environment = environment.environment;
exports.error = error.error;
exports.eventloop = eventloop.eventloop;
exports.func = _function._function;
exports.indexeddb = indexeddb.indexeddb;
exports.iterator = iterator.iterator;
exports.json = json.json;
exports.logging = log__namespace;
exports.map = map.map$1;
exports.math = math.math;
exports.mutex = mutex.mutex;
exports.number = number.number;
exports.object = object.object;
exports.pair = pair.pair;
exports.prng = prng.prng;
exports.promise = promise.promise;
exports.set = set.set;
exports.sort = sort.sort;
exports.statistics = statistics.statistics;
exports.string = string.string;
exports.symbol = symbol.symbol;
exports.time = time.time;
exports.tree = tree.tree;
exports.websocket = websocket.websocket;
