// lib0/dist/string.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

require('./array-f0f52786.cjs');
var string = require('./string-3cd7a2db.cjs');
require('./set-a0a3ea69.cjs');



exports.MAX_UTF16_CHARACTER = string.MAX_UTF16_CHARACTER;
exports._decodeUtf8Native = string._decodeUtf8Native;
exports._decodeUtf8Polyfill = string._decodeUtf8Polyfill;
exports._encodeUtf8Native = string._encodeUtf8Native;
exports._encodeUtf8Polyfill = string._encodeUtf8Polyfill;
exports.decodeUtf8 = string.decodeUtf8;
exports.encodeUtf8 = string.encodeUtf8;
exports.fromCamelCase = string.fromCamelCase;
exports.fromCharCode = string.fromCharCode;
exports.fromCodePoint = string.fromCodePoint;
exports.repeat = string.repeat;
exports.splice = string.splice;
exports.trimLeft = string.trimLeft;
exports.utf8ByteLength = string.utf8ByteLength;
Object.defineProperty(exports, 'utf8TextDecoder', {
	enumerable: true,
	get: function () { return string.utf8TextDecoder; }
});
exports.utf8TextEncoder = string.utf8TextEncoder;
